src/app/components/form/form.component.ts

```diff
@@ -137,5 +136,7 @@ export class FormComponent implements OnInit {
       })
   }
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    localStorage.clear()
```

Comment:
Why?

Reply:
becuse if i reomove this so the data from previus scouts stays there

Reply:
If the user's app crashes and comes back again they should have the previous scout indeed...